frontend/apps/zk-poker/src/features/wallet/varan-wallet.ts

```diff
@@ -0,0 +1,189 @@
+import { InjectedAccount, InjectedWindow } from '@polkadot/extension-inject/types';
```

Comment:
Let's not commit any code related to Varan Wallet as of now, since there's a hope that it's team will implement API according to Substrate standards. Are we able to use `@gear-js/wallet-connect` without it?